wurst/_wurst/Assets.wurst

```diff
@@ -16222,3 +16222,380 @@ public class Icons
 	static constant pASBTNVampiricAura = "ReplaceableTextures\\PassiveButtons\\PASBTNVampiricAura.blp"
 	static constant pASBTNVorpalBlades = "ReplaceableTextures\\PassiveButtons\\PASBTNVorpalBlades.blp"
 	static constant pASBTNWarStomp = "ReplaceableTextures\\PassiveButtons\\PASBTNWarStomp.blp"
+
+
+/*
+	Source: cj_order.j v 0.12
```

Comment:
I don't think this belongs into "assets", since they aren't actually resources.
I think this can be put into a separate package.

Reply:
Hm... Where should it be placed then?